api/src/db/DynamoBusinessesDataClient.ts

```
@@ -101,19 +97,16 @@ export const DynamoBusinessesDataClient = (
         businessId: businessId,
       },
     };
-    return db
-      .send(new GetCommand(params))
-
-      .then((result) => {
-        if (!result.Item) {
-          throw new Error("Not found");
-        }
-        return doMigration(result.Item.data);
-      });
+    return db.send(new GetCommand(params)).then((result) => {
+      if (!result.Item) {
+        throw new Error("Not found");
+      }
+      return result.Item.data;
+    });
   };
 
   const put = async (businessData: Business): Promise<Business> => {
-    const migratedData = migrateData(businessData);
+    const migratedData = migrateData(businessData, logger);
```

Why is the migrate data function necessary on the put statement?

---

It should throw an error b/c migrateData expects userData